web/pdf_viewer.js

```diff
@@ -128,6 +128,7 @@ function isValidAnnotationEditorMode(mode) {
  *   rendering. The default value is `false`.
  * @property {boolean} [supportsPinchToZoom] - Enable zooming on pinch gesture.
  *   The default value is `true`.
+ * @property {boolean} [enableAutoLinking] - Enable automatic linking of URLs.
```

Comment:
This is perhaps not all that helpful if you don't already know what the feature does.
```suggestion
 * @property {boolean} [enableAutoLinking] - Enable creation of hyperlinks from
     text that look like URLs. The default value is `false`.
```